src/plugins/geomap/demo/config-fr.js

```diff
@@ -123,7 +132,7 @@ var wet_boew_geomap = {
 						"7yU8DvztaN22EZIXSWIdSf30jRE5OtjylklG/36+y5Wx6YaX0Z16e24i/AWg1hRJKTWppAAAAAElFTkSuQmCC",
 						fillOpacity: "1",
 						graphicWidth: "25",
-						name: "Mines de non-métaux"
+						name: "Nonmetal Mines"
```

Comment:
Do we need to use the English name? Can we keep the French name?

Reply:
Fixed in 27caeb5aa2b3fef1718e261c90ebec24dba2c448